experimental/auth-logic/run_auth_logic.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2022 The Project Oak Authors
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+  "fmt"
+  "os/exec"
+  "io/ioutil"
+  "strings"
+)
+
+func process_command(cmd *exec.Cmd) {
+  _, err := cmd.Output()
+
+  if err != nil {
+        fmt.Println(err)
```

Comment:
I think it is more conventional to use `log.Fatalf` with a custom error message context. For instance like `log.Fatalf("Couldn't process command: %v", err)`.

Also, it is move conventional to combine the `if` with the previous statement:

```
  if  _, err := cmd.Output(); err != nil {
        log.Fatalf("Couldn't process command: %v", err)
  }
```